vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt

```diff
@@ -42,8 +42,9 @@ import org.matrix.android.sdk.api.session.room.failure.CreateRoomFailure
 import org.matrix.android.sdk.api.session.room.model.RoomDirectoryVisibility
 import org.matrix.android.sdk.api.session.room.model.create.CreateRoomParams
 import org.matrix.android.sdk.api.session.room.model.create.CreateRoomPreset
+import timber.log.Timber
 
-class CreateRoomViewModel @AssistedInject constructor(@Assisted initialState: CreateRoomViewState,
+class CreateRoomViewModel @AssistedInject constructor(@Assisted val initialState: CreateRoomViewState,
```

Comment:
`initialState` could be private (I think)

Reply:
done